lambdas/tests/unit/helpers/data/statistic/mock_logs_query_results.py

```diff
@@ -2,19 +2,24 @@
 USER_ID_2 = "9E7F1235-3DF1-4822-AFFB-C4FCC88C2690"
 HASHED_USER_ID_1 = "3192b6cf7ef953cf1a1f0945a83b55ab2cb8bae95cac6548ae5412aaa4c67677"
 HASHED_USER_ID_2 = "a89d1cb4ac0776e45131c65a69e8b1a48026e9b497c94409e480588418a016e4"
+HASHED_USER_ID_1_WITH_ADMIN_ROLE = HASHED_USER_ID_1 + " - GP_ADMIN - RO76"
+HASHED_USER_ID_1_WITH_PCSE_ROLE = HASHED_USER_ID_1 + " - PCSE - "
+HASHED_USER_ID_2_WITH_CLINICAL_ROLE = HASHED_USER_ID_2 + " - GP_CLINICAL - RO76"
```

Comment:
I suggest using f-strings here for improved readability.
```suggestion
HASHED_USER_ID_1_WITH_ADMIN_ROLE = f"{HASHED_USER_ID_1} - GP_ADMIN - RO76"
HASHED_USER_ID_1_WITH_PCSE_ROLE = f"{HASHED_USER_ID_1} - PCSE - "
HASHED_USER_ID_2_WITH_CLINICAL_ROLE = f"{HASHED_USER_ID_2} - GP_CLINICAL - RO76"
```